utilities/src/main/java/io/onetable/utilities/RunSync.java

```diff
@@ -141,10 +141,8 @@ public static void main(String[] args) throws IOException {
         ReflectionUtils.createInstanceOfClass(sourceProviderClass);
     sourceClientProvider.init(hadoopConf, sourceClientConfig.configuration);
 
-    List<io.onetable.model.storage.TableFormat> tableFormatList =
-        datasetConfig.getTargetFormats().stream()
-            .map(TableFormat::valueOf)
-            .collect(Collectors.toList());
+    List<String> tableFormatList =
+        datasetConfig.getTargetFormats().stream().map(String::valueOf).collect(Collectors.toList());
```

Comment:
I think we can just directly use the `datasetConfig.getTargetFormats()` now since that returns a list of strings.

Reply:
done.